sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -128,6 +129,15 @@ case class CreateViewCommand(
     qe.assertAnalyzed()
     val analyzedPlan = qe.analyzed
 
+    // CREATE VIEW AS INSERT INTO ... is not allowed, we should throw an AnalysisException.
+    analyzedPlan match {
+      case i: InsertIntoHadoopFsRelationCommand =>
```

Comment:
shall we forbid all commands? e.g. `CREATE VIEW xxx AS CREATE TABLE ...` should also be disallowed right?